ipam/calico-ipam.go

```diff
@@ -14,24 +14,24 @@
 package main
 
 import (
+	"context"
 	"encoding/json"
 	"flag"
 	"fmt"
 	"net"
-
 	"os"
 
-	log "github.com/sirupsen/logrus"
-
 	"github.com/containernetworking/cni/pkg/skel"
-	"github.com/containernetworking/cni/pkg/types"
+	cniTypes "github.com/containernetworking/cni/pkg/types"
 	"github.com/containernetworking/cni/pkg/types/current"
 	cniSpecVersion "github.com/containernetworking/cni/pkg/version"
+	"github.com/projectcalico/cni-plugin/types"
```

Comment:
maybe have this as calicotypes too, to avoid confusion?

Reply:
actually, it looks like this change is because you moved a file from utils to types.

IMHO both utils and types are rather meaningless package names so I don't think this rename has actually done anything other than cause a conflict over two different type packages.  I think this would have been best left unchanged.

Reply:
I prefer not to unless you have a strong preference

Reply:
ditto.

Reply:
yeah i'm not overly fussed.